docs/content/uijson_format/json_objects.rst

```diff
@@ -0,0 +1,299 @@
+max ``float``
+    Optional maximum value allowed for the value. The default is the maximum numeric limits of float.
+precision ``int``
+    Optional number of decimal places for the value.
+
+
+Drop-down data parameter
+------------------------
+In this example, the object parameter *data_mesh* is also given for reference.
+
+.. code-block:: json
+
+   {
+   "data_channel": {
+   "main": true,
+   "association": "Vertex",
+   "dataType": "Float",
+   "label": "Data channel",
+   "parent": "data_mesh",
+   "value": ""
+   },
+   "data_mesh": {
+   "main": true,
+    "meshType": ["{202C5DB1-A56D-4004-9CAD-BAAFD8899406}" ,
+      "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}"],
+   "main": true,
+   "label": "Select Points or Curve",
+   "value": ""
+   }
+   }
+
+
+.. figure:: ./images/data_param.png
+
+
+
+Data or value parameter
+-----------------------
+In some cases, a simple float will do. The use of the member **isValue** and **property** together allows the choice within the UI. In this example, the **property** member will turned on to begin with. The **value** will default to 1.0 and show up when the icon is clicked. The **uncertainty channel** object depends on the **data_mesh** object. The combobox will filter data from the chosen object that is located on the vertices and is float (pi icon).
+
+
+.. code-block:: json
+
+   {
+   "uncertainty_channel": {
+   "main": true,
+   "association": "Vertex",
+   "dataType": "Float",
+   "isValue": false,
+   "property": "",
+   "min": 0.001,
+   "label": "Uncertainty",
+   "parent": "data_mesh",
+   "value": 1.0
+   },
+   "data_mesh": {
+   "main": true,
+    "meshType": ["{202C5DB1-A56D-4004-9CAD-BAAFD8899406}" ,
+      "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}"],
+   "main": true,
+   "label": "Select Points or Curve",
+   "value": ""
+   }
+   }
+
+
+.. figure:: ./images/data_value_param.png
+
+
+
+Dependencies on other parameters
+================================
+
+Use the **dependency** and **dependencyType** members to create dependencies. The parameter driving the dependency should have the **optional** member or be a :ref:`Boolean parameter'<bool_param>`. Below are a couple of examples. The first requires the checkbox to be checked prior to the string parameter to be enabled. When the dependency is checked, the dependent parameter is *enabled*.
+
+
+.. code-block:: json
+
+   {
+   "python_interest": {
+   "main": true,
+   "label": "Do you like Python?",
+   "value": false,
+   "tooltip": "Check if you like Python"
+   },
+   "favourite_package": {
+   "main": true,
+   "label": "Favourite Python package",
+   "value": "geoh5py",
+   "dependency": "python_interest",
+   "dependencyType": "enabled"
+   }
+   }
+
+
+.. figure:: ./images/dependency_ex1.png
+
+
+The next example has a dependency on an optional parameter. The member *enabled* is set to false so that it is not automatically checked. This is a parameter that Geoscience ANALYST will update on export.
```

Comment:
```suggestion
The next example has a dependency on an optional parameter. The **enabled** field of the dependency is set to false so that it is not automatically checked. When the ``territory`` checkbox is selected, both the ``territory`` and ``city`` parameters will become enabled. Geoscience ANALYST will update this parameter on export.
```